WorkflowLayer/WorkflowLayer.csproj

```diff
@@ -52,50 +52,59 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Bio.Core, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\NetBio.Core.3.0.0-alpha\lib\netstandard2.0\Bio.Core.dll</HintPath>
+    <Reference Include="Bio.Core, Version=2.0.0.1, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\NetBioCore.PCL.2.0.150722\lib\net45\Bio.Core.dll</HintPath>
+    </Reference>
+    <Reference Include="Bio.Desktop, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\NetBioCore.PCL.2.0.150722\lib\net45\Bio.Desktop.dll</HintPath>
+    </Reference>
+    <Reference Include="Bio.Platform.Helpers, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\NetBioCore.PCL.2.0.150722\lib\net45\Bio.Platform.Helpers.dll</HintPath>
+    </Reference>
+    <Reference Include="Bio.VCF, Version=0.0.55.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Bio.VCF.ajc.0.0.55\lib\net461\Bio.VCF.dll</HintPath>
     </Reference>
     <Reference Include="Bio.VCF.ajc, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Bio.VCF.ajc.0.0.56\lib\net471\Bio.VCF.ajc.dll</HintPath>
+      <HintPath>..\packages\Bio.VCF.ajc.0.0.57\lib\netstandard2.0\Bio.VCF.ajc.dll</HintPath>
```

Comment:
I'm glad to see this changed... this sort of thing has been the bane of my existence the last two days